### PR TITLE
Correct type annotation for `inspect` to match README

### PR DESCRIPTION
**Pull request by erickzhao**
### Initial checklist

* [x] I read the support docs 
* [x] I read the contributing guide 
* [x] I agree to follow the code of conduct 
* [x] I searched issues and discussions and couldn’t find anything or linked relevant results below 
* [x] I made sure the docs are up to date
* [x] I included tests (or that’s not needed)

### Description of changes

The description of the `inspect` function in the type annotation does not match the description in the README and is misleading.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const emptyOptions = {}
 const own = {}.hasOwnProperty
 
 /**
- * Inspect a node, without color.
+ * Inspect a tree, with color in Node, without color in browsers.
```
Comment (wooorm) [inline lib/index.js:16]:
```suggestion
 * Inspect a tree.
```

- Agree that `tree` is better.
- `, without color.` is of course wrong, but thinking about this again, I think the `with color in Node, without color in browsers.` is not needed? It’s about the defaults, but that isn’t mentioned.

**Review by wooorm — COMMENTED** [review]
Good catch!

File: lib/index.js (hunk 1 above)
Comment (erickzhao) [reply lib/index.js]: I think that makes a lot of sense! Here's a commit to address that: [8ef1255](https://github.com/syntax-tree/unist-util-inspect/pull/22/commits/8ef1255fbc714eb4f3b3e1a6b00f187d7206b2c1)

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
Thank you! :)

**Pull request merged**